packages/circuit-ui/components/RadioButton/RadioButton.spec.tsx

```diff
@@ -69,6 +69,21 @@ describe('RadioButton', () => {
       const inputEl = screen.getByRole('radio');
       expect(inputEl).toHaveAccessibleName(defaultProps.label);
     });
+
+    it('should have a helper text', () => {
+      const ref = createRef<HTMLInputElement>();
+      render(
+        <RadioButton
+          ref={ref}
+          {...defaultProps}
+          helperText="Some explanation"
+        />,
+      );
```

Comment:
The `ref` isn't needed in this test.
```suggestion
      render(
        <RadioButton
          {...defaultProps}
          helperText="Some explanation"
        />,
      );
```